packages/electrumAnalyzer/electrumAnalyzer.0.3.5-2/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+maintainer: "david.chemouil+electrum@onera.fr"
+authors: ["David Chemouil" "Julien Brunel" "Denis Kuperberg"]
+homepage: "https://forge.onera.fr/projects/electrum"
+bug-reports: "https://forge.onera.fr/projects/electrum"
+license: "GPL-3+"
+tags: "org:onera"
+dev-repo: "https://github.com/grayswandyr/electrumAnalyzer.git"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+build-test: [
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-test"]
+]
+remove: [ [  "ocaml" "%{etc}%/electrumAnalyzer/_oasis_remove_.ml" "%{etc}%/electrumAnalyzer" ]
+]
+depends: [
+  "batteries" {build}
+  "cmdliner" {build}
+  "menhir" {build & >= "20150720"}
+  "ocamlfind" {build}
+  "pprint" {build}
```

Comment:
I see the PR was merged so I guess I don't have to do anything right now. For future releases, am I expected to add `ocamlbuild {build}` or something else in the depends field?
(I wish the whole OCaml build and release experience was smoother, at least for pure OCaml projects like this one; when you're not used to releasing packages more than once or twice a year, it's easy to forget some parts of the workflow).